### PR TITLE
first-time-install installs browsers

### PR DESCRIPTION
**Pull request by sinisastanic**

### CODE REVIEW

File: src/debugtopus.ts [hunk 1]
```diff
@@ -101,6 +102,25 @@ export const runTest = async ({
...
+      const someVariable = promisify(exec)("npx playwright install", {
+        cwd: packageRootDir,
+      });
+
+      someVariable.child.stdout!.on("data", (data) => console.log(data));
+
+      await someVariable; // todo:  this does not work
+    }
+  });
+
```
Comment (sinisastanic) [inline src/debugtopus.ts:123]: I had it working as expected like this

```typescript
  const file = chromium.executablePath();
  try {
    await fs.access(file, fs.constants.X_OK, );
  } catch (e) {
    const error = e as Record<string, string>;
    if ("code" in error && error.code === "ENOENT") {
      console.log(
        "Couldn't find chromium executable, some message about running 'npx playwright install'..."
      );
      const someVariable = promisify(exec)("npx playwright install", {
        cwd: packageRootDir,
      });
      someVariable.child.stdout!.on("data", (data) => console.log(data));

      await someVariable;
    }
  }
```

but I like above code more, I just don't know how to await "error promise"

File: src/debugtopus.ts [hunk 2]
```diff
@@ -101,6 +102,25 @@ export const runTest = async ({
...
```
Comment (sinisastanic) [inline src/debugtopus.ts:123]: I had it working as expected like this

```typescript
  const file = chromium.executablePath();
  try {
    await fs.access(file, fs.constants.X_OK, );
  } catch (e) {
    const error = e as Record<string, string>;
    if ("code" in error && error.code === "ENOENT") {
      console.log(
        "Couldn't find chromium executable, some message about running 'npx playwright install'..."
      );
      const someVariable = promisify(exec)("npx playwright install", {
        cwd: packageRootDir,
      });
      someVariable.child.stdout!.on("data", (data) => console.log(data));

      await someVariable;
    }
  }
```

but I like commited code more, I just don't know how to await "error promise"

**Comment by sinisastanic** [comment]
version bump missing 🙈

File: src/debugtopus.ts [hunk 3]
```diff
@@ -76,18 +77,53 @@ export const prepareTestRun = async ({
...
   writeFileSync(testFilePath, code);
 
-  const configFilePath = path.join(tempDir, `${randomUUID()}.config.ts`);
+  const configFilePath = path.join(tempDir, `${fileNameUUID}.config.ts`);
   writeFileSync(configFilePath, getConfig(url, outputDir));
 
   return { testFilePath, configFilePath, outputDir, packageRootDir };
 };
 
+async function ensureChromiumIsInstalled(packageRootDir: string) {
```
Comment (Germandrummer92) [inline src/debugtopus.ts:92]: arrow function please and return type 😬

Comment (Germandrummer92) [reply src/debugtopus.ts:92]: and some tests please

Comment (Germandrummer92) [reply src/debugtopus.ts:92]: and adjust the docs in https://github.com/OctoMind-dev/mintlify-docs as well so it doesn't have the "error" anymore ;)

**Review by Germandrummer92 — APPROVED** [review]

**Pull request merged**